packages/vc-export/src/fromICredential.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2018-2023, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import { DidUri, ICredential, ICType } from '@kiltprotocol/types'
```

Comment:
Should these be type imports?

Reply:
9a0ded49479e5153709c2e287bdb23df3ed15731